code/modules/modular_computers/file_system/programs/app_presets.dm

```diff
@@ -247,6 +247,31 @@
 	)
 	return _prg_list
 
+/datum/modular_computer_app_presets/security/armory
+	name = "security_arm"
+	display_name = "Security - Armory"
+	description = "Contains the most common security and armory programs."
+	available = FALSE
+
+/datum/modular_computer_app_presets/security/armory/return_install_programs(obj/item/modular_computer/comp)
+	var/list/_prg_list = list(
+		new /datum/computer_file/program/nttransfer(comp),
+		new /datum/computer_file/program/newsbrowser(comp),
+		new /datum/computer_file/program/manifest(comp),
+		new /datum/computer_file/program/filemanager(comp),
+		new /datum/computer_file/program/chatclient(comp),
+		new /datum/computer_file/program/civilian/cargoorder(comp),
+		new /datum/computer_file/program/camera_monitor(comp),
+		new /datum/computer_file/program/comm(comp),
+		new /datum/computer_file/program/digitalwarrant(comp),
+		new /datum/computer_file/program/penal_mechs(comp),
+		new /datum/computer_file/program/records/security(comp),
+		new /datum/computer_file/program/guntracker(comp),
+		new /datum/computer_file/program/implant_tracker(comp),
+		new /datum/computer_file/program/ntsl2_interpreter(comp)
```

Comment:
Does security need or use NTSL2 interpreter? Are they even allowed to tcomms? Or is it used somewhere else?

Reply:
yeah, it's like an in-built circuitry program, not necessarily connected to tcomms.